src/sentry/static/sentry/app/views/eventsV2/table/tableActions.tsx

```diff
@@ -90,6 +90,15 @@ renderAsyncExportButton.propTypes = {
   isLoading: PropTypes.bool,
 };
 
+function renderSummaryButton() {
```

Comment:
@doralchan if you're looking for feedback; it looks fine design-wise.

Might want to make it more functional though :D